examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,202 @@
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+            // Add a menu :
+            let menuGlobe = new GuiTools('menuDiv', view);
+            // Add an option in the menu to display picked position as 3D axes :
+            const displayPositions = menuGlobe.addGUI('PickedPositions', false, function (v) {
+                if (v) {
+                    view.scene.add(pickedPointsGroup);
+                    view.notifyChange();
+                } else {
+                    view.scene.remove(pickedPointsGroup);
+                    view.notifyChange();
+                }
+            })
+
+            // THREE.Group which stores picked points markers.
+            let pickedPointsGroup = new itowns.THREE.Group();
+
+            // Array which stores picked camera positions, headings, tilts...
+            let cameraTravel = [];
+
+            let isFirstPickedPoint = true;
+            let travelOn = 0;
+
+            // Add a pointer at a given position.
+            function addMeshToScene(position) {
+                // Creation of the new mesh :
+                let mesh = new itowns.THREE.AxesHelper(50000);
+                mesh.position.copy(position.as(view.referenceCrs));
+                mesh.lookAt(position.geodesicNormal);
+                mesh.rotateY(Math.PI);
+                mesh.updateMatrixWorld();
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearMeshes() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Utility method to compute the positive mod.
+            Number.prototype.mod = function(n) {
+                const m = ((this % n) + n) % n;
+                return m < 0 ? m + Math.abs(n) : m;
+            }
+
+            // Listens for user input :
+            window.addEventListener('keypress', async function (event) {
+
+                // If user presses `s` (for 'save'), saves the current camera position, heading, tilt...
+                if (event.key === 's') {
+                    // Get camera coordinates :
+                    let cameraCoord3D = new itowns.Coordinates(view.referenceCrs);
+                    cameraCoord3D.setFromVector3(view.camera.camera3D.position);
+
+                    // Saves camera current data :
+                    let pathStep = {
+                        coord: view.controls.getLookAtCoordinate().as(view.referenceCrs),
+                        coord3D: cameraCoord3D,
+                        range: view.controls.getRange(),
+                        heading: view.controls.getHeading(),
+                        tilt: view.controls.getTilt(),
+                    };
```

Comment:
could you use `CameraUtils.getTransformCameraLookingAtTarget` to get all properties?

Reply:
I now use `CameraUtils.getTransformCameraLookingAtTarget` to get all properties.